vfit.1

```diff
@@ -0,0 +1,31 @@
+.TH vfit "1" "December 2021" "User Commands"
+.SH NAME
+vfit \- generate backwards-compatible, static instances of variable fonts
+.SH SYNOPSIS
+vfit [\-h] [\-o PATH] [\-f {woff,woff2}] [\-C] config source
+.SH DESCRIPTION
+This (Variable Font Instancing Tool) allows you to generate custom, static
+instances of a variable font defined in a configuration file.
+.SH OPTIONS
+.PP
+Generate backwards\-compatible, static instances of variable fonts.
+.SS "positional arguments:"
+.TP
+config
+the metadata/style definition file
+.TP
+source
```

Comment:
I believe "config" and "source" should be bold (like the flags) so they are more easily distinguishable from their description text?
<img width="405" alt="image" src="https://user-images.githubusercontent.com/41548458/146383867-d061c7fa-ea8f-490f-95ed-142d3b5133a6.png">